docs/en/glossary/glossary.asciidoc

```diff
@@ -358,6 +369,11 @@ include::{es-repo-dir}/glossary.asciidoc[tag=document-def]
 include::{kib-repo-dir}/glossary.asciidoc[tag=edge-def]
 --
 
+[[glossary-{agent}]] {agent}::
```

Comment:
The definition seems to apply to Elastic Agent, not just any agent.

Reply:
That's correct, `{agent}` resolves to Elastic Agent. Would you like to see an additional entry for `agent`?

Reply:
that works thanks!